web/src/main/java/com/graphhopper/countries/GermanySpatialRule.java

```diff
@@ -15,17 +15,18 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-package com.graphhopper.routing.util.spatialrules.countries;
+package com.graphhopper.countries;
```

Comment:
It's not in the core, it's in the web module, and it's just an example. I don't know where it will be in the end.